init/atboot.pl

```diff
@@ -12,7 +12,7 @@
 
 if ($init_mode eq "osx") {
 	# Darwin System
-	&enable_at_boot("webmin", "Webmin administration server",
+	&enable_at_boot("$product", "Webmin administration server",
```

Comment:
This can be just $product instead of "$product"

Reply:
Then we need to test that user defined env variable to comply to a certain format .. what if that would be set to "webmin 1.999-4" (intentionally or by mistake?).

Reply:
What I meant was, in Perl there's no difference between `$foo` and `"$foo"`

Reply:
Oh, that's Perl, right. Removed.